src/textField/index.tsx

```diff
@@ -68,9 +68,13 @@ export interface TextFieldState {
 const DisplayError = (error: errorField) => {
   const className = 'kirk-error-message'
 
-  return React.isValidElement(error)
-    ? React.cloneElement(error, { className } as Object)
-    : <span role="alert" className={className}>{error}</span>
+  return React.isValidElement(error) ? (
+    React.cloneElement(error, { className } as Object)
+  ) : (
+    <span role="alert" className={className}>
```

Comment:
I know that you're only re-indenting stuff here, but shouldn't this be a `<Text>` here?
Just sayin' ;D